### PR TITLE
driver actuator ConvertStage: fix moveAbs() going to wrong position when called with only one axis

### PR DESCRIPTION
**Pull request by pieleric**

### CODE REVIEW

File: src/odemis/driver/test/actuator_test.py [hunk 1]
```diff
@@ -507,6 +523,10 @@ def test_move_abs(self):
         f.result()
         test.assert_pos_almost_equal(stage.position.value, {"x": 1e-06, "y": 2e-06})
         test.assert_pos_almost_equal(child.position.value, {"x":-2e-06, "y": 1e-06})
```
Comment (Sabrina85) [inline src/odemis/driver/test/actuator_test.py]: empty space after colon missing (x)

File: src/odemis/driver/test/actuator_test.py [hunk 2]
```diff
@@ -507,6 +523,10 @@ def test_move_abs(self):
...
+        f = stage.moveAbs({"x": 1e-06})  # Test only move only one axis
```
Comment (Sabrina85) [inline src/odemis/driver/test/actuator_test.py]: request current position again for x?

File: src/odemis/driver/test/actuator_test.py [hunk 3]
```diff
@@ -507,6 +523,10 @@ def test_move_abs(self):
...
+        f.result()
+        test.assert_pos_almost_equal(stage.position.value, {"x": 1e-06, "y": 2e-06})
+        test.assert_pos_almost_equal(child.position.value, {"x":-2e-06, "y": 1e-06})
```
Comment (Sabrina85) [inline src/odemis/driver/test/actuator_test.py]: missing space after colon (x)

File: src/odemis/driver/test/actuator_test.py (hunk 1 above)
Comment (Sabrina85) [inline src/odemis/driver/test/actuator_test.py]: question: why are positions for child swapped compared to stage?

File: src/odemis/driver/test/actuator_test.py [hunk 4]
```diff
@@ -494,6 +506,10 @@ def test_move_abs(self):
         f.result()
         test.assert_pos_almost_equal(stage.position.value, {"x": 1e-06, "y": 2e-06})
         test.assert_pos_almost_equal(child.position.value, {"x": 1e-05, "y": 2e-05})
+        f = stage.moveAbs({"x": 1e-06})  # Test only move only one axis
+        f.result()
+        test.assert_pos_almost_equal(stage.position.value, {"x": 1e-06, "y": 2e-06})
+        test.assert_pos_almost_equal(child.position.value, {"x": 1e-05, "y": 2e-05})
```
Comment (Sabrina85) [inline src/odemis/driver/test/actuator_test.py]: question: why are child positions one magnitude less compared to stage?

File: src/odemis/driver/test/actuator_test.py [hunk 5]
```diff
@@ -494,6 +506,10 @@ def test_move_abs(self):
         f.result()
         test.assert_pos_almost_equal(stage.position.value, {"x": 1e-06, "y": 2e-06})
         test.assert_pos_almost_equal(child.position.value, {"x": 1e-05, "y": 2e-05})
+        f = stage.moveAbs({"x": 1e-06})  # Test only move only one axis
```
[the remark this answers is not in the document]
Comment (Sabrina85) [reply src/odemis/driver/test/actuator_test.py]: requesting the same position for x again?

File: src/odemis/driver/test/actuator_test.py (hunk 1 above)
Comment (pieleric) [reply src/odemis/driver/test/actuator_test.py]: Because we are testing with rotation = 90°

File: src/odemis/driver/test/actuator_test.py (hunk 4 above)
Comment (pieleric) [reply src/odemis/driver/test/actuator_test.py]: Because we are testing with scale = 10

**Comment by pieleric** [comment]
Updated the code with longer comment before the moves with only one axis. I won't fix the dict pep8 issues, because this is due to PyDev auto-correcting it. I could fight against that tool, but it's not really worthy.

**Pull request merged**